### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#32920]
#### VFS team name
Identity 

#### VFS product name
Sign-in modal

#### Point of Contact/Reviewers
Allison Christman (@allison0034) - Design

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The body font is too small.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Body font size should have .vads-u-font-size--base class. See https://design.va.gov/utilities/font-size

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
